tiled/client/array.py

```diff
@@ -92,11 +93,13 @@ def _get_block(self, block, dtype, shape, slice=None):
             expected_shape = ",".join(map(str, shape))
         else:
             expected_shape = "scalar"
+        url_path = self.item["links"]["block"]
         content = handle_error(
             self.context.http_client.get(
-                self.item["links"]["block"],
+                url_path,
```

Comment:
I confirmed that in current httpx any query params here are ignored in `params` is passed.

```py
In [6]: httpx.get("http://example.com?a=1&b=3", params={'a': 2, 'c': 3}).request.url
Out[6]: URL('http://example.com?a=2&c=3')

In [7]: httpx.get("http://example.com?a=1&b=3").request.url
Out[7]: URL('http://example.com?a=1&b=3')
```